syntax = "proto3";
package route_server;

message HeartBeatRequest {
    int32 msg = 1;
};

message HeartBeatResponse {
    int32 msg = 1;
    string error_msg = 2;
};

service RouteServer {
    rpc GetClientHeartBeat(HeartBeatRequest) returns (stream HeartBeatResponse) {}
}